tests/framework/pkg/schemas/rke.cattle.io/v1/schema.go

```diff
@@ -0,0 +1,21 @@
+package schema
+
+import (
+	"github.com/rancher/norman/types"
+	v1 "github.com/rancher/rancher/pkg/apis/rke.cattle.io/v1"
+)
+
+var (
+	Version = types.APIVersion{
+		Version: "v1",
+		Group:   "rke.cattle.io",
+		Path:    "/v1",
+	}
+
+	Schemas = RKESchemas(&Version).
+		Init(rkeTypes)
+)
+
+func rkeTypes(schemas *types.Schemas) *types.Schemas {
+	return schemas.MustImport(&Version, v1.RKEControlPlane{})
```

Comment:
Hi Jake, just checking do you not need to have the `schema.ID` set to `"rke.cattle.io" `?

Reply:
Truth be told I'm not sure, this seemed to be the minimal configuration I needed to generate the clients.

Reply:
Truth be told I'm not sure, this seemed to be the minimal configuration I needed to generate the clients.

Reply:
Oh ok. I'm just checking, because for the provisioning client for example, I needed to update the schema id to be  able to hit the expect API endpoint. If that is not the case here, it looks good to me. The tests run fine?

Reply:
Sounds good thank you Jake. Looks go to me